apps/stats/stats-1.py

```diff
@@ -330,16 +323,11 @@ def __printPerformance( self, script, args ) :
 			print "Performance :\n"
 			self.__printItems( self.__timers.items() )
 
-			if self.__performanceMonitor is None :
-				return
-
-			stats = self.__performanceMonitor.allStatistics().items()
-
-			print "\nMost frequently hashed :\n"
-			self.__printStatisticsItems( script, stats, lambda x : x[1].hashCount, args["mostFrequentlyHashed"].value )
-
-			print "\nMost frequently computed :\n"
-			self.__printStatisticsItems( script, stats, lambda x : x[1].computeCount, args["mostFrequentlyComputed"].value )
+			if self.__performanceMonitor is not None :
+				print "\n" + Gaffer.formatStatistics(
```

Comment:
Oh, I thought we had decided in those discussion that we wanted everything moving to the namespaced form (rather than turning the module namespace into a dumping ground of free functions, like IECore has become).

But I'm fine with this for now if we're still aiming to move over at the same time as the c++ side.